wordle-web/layouts/default.vue

```diff
@@ -57,27 +57,17 @@
   </v-app>
 </template>
 
-<script>
-export default {
-  name: 'DefaultLayout',
-
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+@Component({})
+export default class Default  extends Vue{
+name : string ="DefaultLayout";
   data() {
```

Comment:
While the language has been changed to typescript, the file is still written in JavaScript style.

Take a look at the [`game.vue`](https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/blob/37aea99bec3f0e29bbb3e3d4bbf7a4413bbac94b/wordle-web/pages/game.vue#L103) page for an example of a typescript style